core/src/apps/cardano/helpers/protocol_magics.py

```diff
@@ -7,5 +7,9 @@
 }
 
 
+def is_mainnet(protocol_magic: int) -> bool:
+    return protocol_magic == MAINNET
```

Comment:
Protocol magic for mainnet is guaranteed to be only one? In another words the comment in `core/src/apps/cardano/helpers/network_ids.py` does not apply?

Reply:
I don't think it is guaranteed, however, we can't do a `protocol_magic != TESTNET` check with protocol magics because there can be multiple testnet protocol magics.